examples/Image Validation/Readme.md

```diff
@@ -0,0 +1,70 @@
+Dropzone for images validating dimensions and type of dropped images
+
+```
+class BasicValidatedImage extends React.Component {
+  constructor() {
+    super()
+    this.state = { files: [] }
+  }
+
+  onDrop(accepted, rejected, dropZoneRef) => {
+    let errors = [];
+
+    var acceptedImg = accepted[0];
+    if (acceptedImg.type !== 'image/jpeg') {
+      errors.push('This image is not a jpg and is therefore rejected.');
+    }
+    if (acceptedImg.size > 10000000) {
```

Comment:
`10000000` is less than 10MB. Let's replace it with `10 * 1024 * 1024` to make the intent more clear and add a comment.